webmails/roundcube/mailu.js

```diff
@@ -0,0 +1,9 @@
+rcmail.addEventListener('init', function(evt) {
+      var btn = document.createElement('a');
+      btn.setAttribute("href", "/admin");
```

Comment:
You make the assumption that admin is always available on endpoint /admin. This is incorrect. It is defined in environment variable WEB_ADMIN. The path should be derived from this env var.